### DISCUSSION TITLE
How can I let the "keadm init" pull the cloudcore image from a private image registry?

### DISCUSSION THREAD

**Discussion by lengcangche-gituhub** [kubeedge/kubeedge#5957] [Q&A]
In my scenario, there is no Internet connection. I plan to deploy a private registry using Harbor in the Intranet.
I know there is an option "--image-repository" in the "keadm init", however I think this option cannot be used to pull image from a private registry.

**Comment by WillardHu** [comment]
This flag is used to define a private repository for the installation-package image. The pause image needs to be set in the configuration file of the container component. Other image repositories can be set through the resource yaml.

**Reply by lengcangche-gituhub** [reply]
> This flag is used to define a private repository for the installation-package image. The pause image needs to be set in the configuration file of the container component. Other image repositories can be set through the resource yaml.

Thank you. However I don't know where to find the resource yaml. I know keadm is based on helm chart.
